site/releases/2024-dec-24/release-notes.qmd

```diff
@@ -0,0 +1,413 @@
+
+<!---
+PR #1092: [SC-7471] adding custom field permission check
+URL: https://github.com/validmind/frontend/pull/1092
+Labels: enhancement
+--->
+#### Granular role permissions for inventory fields
+
+::: {.column-margin}
+![Setup for an inventory field only editable by developers](developer-only-field.png){fig-alt="A screenshot of the setup for an inventory field only editable by developers" .screenshot}
+:::
+
+:::: {.flex .flex-wrap .justify-around}
+
+::: {.w-70-ns}
+You can now gate edit access to specific model inventory fields to certain roles via the **[write permissions]{.smallcaps}** setting.
+
+:::
+
+::: {.w-30-ns .tc}
+[Manage model inventory fields](/guide/model-inventory/manage-model-inventory-fields.qmd){.button}
+
+:::
+
+::::
+
+- Leave this field blank to leave permissions as default.
+- By default, model owners and validators have write access. Model developers have read-only access.[^4]
+
+## Bug fixes
+
+### {{< var validmind.platform >}} (v1.27.17)
+<!---
+PR #1106: Deleting a finding should remove it from a validation report
+URL: https://github.com/validmind/frontend/pull/1106
+Labels: bug
+--->
+
+:::: {.flex .flex-wrap .justify-around}
+
+::: {.w-60-ns}
+- Previously, deleting a finding did not remove it from linked validation reports. 
+- Now, deleting a finding will show a confirmation warning and subsequently remove it from any linked reports.
+
+::: {.tc}
+[Add and manage model findings](/guide/model-validation/add-manage-model-findings.qmd#delete-model-findings){.button}
+
+:::
+
+:::
+
+::: {.w-40-ns .tc}
+![Deletion confirmation dialogue for a finding linked to a validation report](delete-linked-finding.png){fig-alt="A screenshot of the deletion confirmation dialogue for a finding linked to a validation report" .screenshot}
+
+:::
+
+::::
+
+## Documentation 
+
+### Code samples
+
+#### Initialize {{< var vm.product >}} with stored credentials
+
+We've included details on how to initialize {{< var vm.product >}} with credentials stored in an `.env` file in our Jupyter Notebook samples.
+
+:::: {.flex .flex-wrap .justify-around}
+
+::: {.w-60-ns}
+Our documentation guides have also been updated to match this new experience:
+
+:::
+
+::: {.w-40-ns .tc}
+[Store model credentials in `.env` files](/developer/model-documentation/store-credentials-in-env-file.qmd){.button}
+
+:::
+
+::::
+
+<!---
+PR #543: Add developer fundamentals videos & training fixes
+URL: https://github.com/validmind/documentation/pull/543
+Labels: documentation
+--->
+### User guide updates
+
+#### Developer Fundamentals video series
+
+We created a series of 10 short videos to help you learn about the model documentation process as a developer:
+
+- These videos introduce you to generating model documentation that includes tests. 
+- You will also learn about adding your own tests, editing the content online, and then submitting your documentation for approval so that a validator can review your submitted documentation.
```

Comment:
is it possible to list the 10 videos with a direct link/thumbnail to each? (in the preview, it currently shows only the video #10)

Reply:
I think Nik made this as a playlist, so I will defer to his decision here. @nrichers 

Also, this is linked elsewhere (these are just the releaser notes version).

Reply:
Actually, I just checked the playlist. The issue isn't on our docs end, it's on the YouTube end — it looks like the 10th video is slotted in as the 1st one which is why the thumbnail is like that. @nrichers I don't have access to edit the playlist, can you please help?


| 1st video — 10 | 10th video — 1 |
|---|---|
|![Screenshot 2024-12-20 at 9 34 06 AM](https://github.com/user-attachments/assets/02a3379a-64fd-452a-9148-6b446aa93351) | ![Screenshot 2024-12-20 at 9 34 06 AM](https://github.com/user-attachments/assets/27f3b0d1-e49a-4b20-a9ba-c02dcf02ed3b) |

Once these are swapped then we shouldn't have to display all 10 videos as that will resolve the issue with the 10th video showing up as the thumbnail/1st play.

Reply:
@validbeck the playlist should be fixed now, both in the release notes and the "Guides" section where the video appears:

![image](https://github.com/user-attachments/assets/c60a5040-3e05-4dc7-ab12-6361c0c8b360)

YouTube defaults to putting the latest published video first which is the reverse order in which the videos were uploaded; to fix, I changed the default sort order.

Reply:
Whoo!